src/Makefile.boot

```diff
@@ -23,7 +23,7 @@ INCLUDE_PATHS = \
 
 CACHE_DIR?=./.cache.boot
 
-FSTAR_BOOT ?= $(FSTAR)
+FSTAR_BOOT=fstar.exe
```

Comment:
This is breaking the intended use of `FSTAR_BOOT`, which is to use a different F* binary (from a different tree) to bootstrap the current tree.

Reply:
Thanks, reverted.